### PR TITLE
Add new cli install instructions

### PR DESCRIPTION
**Pull request by gleb-lobov**

### CODE REVIEW

File: docs/understanding-tracking-design/managing-your-data-structures/cli/index.md [hunk 1]
```diff
@@ -72,6 +84,8 @@ Snowplow CLI can take its configuration from a variety of sources. More details
   </TabItem>
 </Tabs>
 
+Snowplow CLI defaults to yaml format. It can be changed to json by either providing a `--output-format json` flag or setting the `output-format: json` config value. It will work for all commands where it matters, not only for `generate`
```
Comment (cksnp) [inline docs/understanding-tracking-design/managing-your-data-structures/cli/index.md:87]: Missing dot at the end.

File: docs/understanding-tracking-design/managing-your-data-structures/cli/index.md [hunk 2]
```diff
@@ -17,19 +17,31 @@ The `data-structures` subcommand of [Snowplow CLI](https://github.com/snowplow-p
...
+brew install snowplow-product/taps/snowplow-cli
+```
+
+Verify the installation with
+```
+snowplow-cli --help
+```
 
-For systems with `curl` available the following commands should get you started with the latest version. Take care to replace `darwin_arm64` with the correct architecture for your system.
+For systems where homebrew is not available binaries for multiple platforms can be found in [releases](https://github.com/snowplow-product/snowplow-cli/releases)
```
Comment (cksnp) [inline docs/understanding-tracking-design/managing-your-data-structures/cli/index.md:32]: Missing dot at the end.

**Review by cksnp — APPROVED** [review]

**Review by cksnp — APPROVED** [review]
I had approved already! 
:-D

**Comment by gleb-lobov** [comment]
@cksnp I don't have access to merge, so you have to click the button :)

**Pull request merged**